proposals/0007-rn-versions-lifecycle-support.md

```diff
@@ -0,0 +1,69 @@
+# RFC: React Native Versions Lifecycle & Support
+
+
+
+>As we are finding ways to better support React Native community, we are moving towards supporting a limited amount of versions of React Native better.
+
+
+The React Native Core team is working to strike a balance between the speed of innovation with the amount of regular change to the platform. We understand that each release brings new value and has an adoption cost to the community.
+This document describes how we intend to formally balance these concerns and provide clear and consistent support over time.
+React Native team will work on best-effort bases to release a new version of React Native every four months. And will focus on speeding up the release cycle and put considerable effort into ensuring a seamless upgrade path.
+This document describes the level of support we can provide and which set of versions is supported.
+
+## Glossary
+
+Below we are defining terms used in this document:
+
+* **stable** **version** - Any version that doesn’t have a -alpha, -beta, -RC postfix in the version name.
+* **latest version** - **** The latest stable version with the highest version number (e.g. 0.68.2).
+    * The highest version number is defined following the [NPM server algorithm](https://github.com/npm/node-semver).
+* **next version** - The next version that hasn’t been fully released yet, and is currently in development/testing (e.g. 0.69.0-RC1).
+* **minor series** - A collection of version (either stable or not stable) sharing the same minor version (e.g. **0.68**.0-RC0, **0.68**.0, **0.68**.1 are all part of the **0.68** minor series).
+
+## Which versions are currently supported?
+
+We’re supporting the **latest** **version**, and the latest versions from the **two previous minor series.**
+We’re also supporting the **next version,** having in mind this is the not yet stable version we are developing at that moment in time.
+
+Versions:
```

Comment:
I think that this should be a living table, so people can refer to that when they need to upgrade or take decisions. I guess that, as soon as this proposal is approved, we would write a SLA page in the website, am I right?

Reply:
Yup this RFC is mostly for discussion and transparency. As we merge it, we can convert it to a page.